contract/name/name.go

```diff
@@ -168,9 +165,7 @@ func registerOwner(scs *state.ContractState, name, owner, destination []byte) er
 }
 
 func setNameMap(scs *state.ContractState, name []byte, n *NameMap) error {
-	lowerCaseName := strings.ToLower(string(name))
-	key := append(prefix, lowerCaseName...)
-	return scs.SetData(key, serializeNameMap(n))
+	return scs.SetData(schema.KeyName(name), serializeNameMap(n))
```

Comment:
Using only `Name` is not clear enough. What about `AccountName`?
```go
return scs.SetData(dbkey.AccountName(name), serializeNameMap(n))
```

Reply:
it is not only about account name, and also include "aergo.name".
it is key function for Name service in governance, so AccountName is Inappropriate.

Reply:
maybe, GovernanceNameKey can be more clear. ( but it should change all governance key like enterprise, system, vote, staking )